src/file/paragraph/properties.ts

```diff
@@ -148,8 +140,8 @@ export class ParagraphProperties extends IgnoreIfEmptyXmlComponent {
             this.push(new TabStop(TabStopType.LEFT, options.leftTabStop));
         }
 
-        if (options.bidirectional) {
-            this.push(new Bidirectional());
+        if (options.bidirectional !== undefined) {
```

Comment:
any reason for this to check for `!== undefined`? is it not more concise to simply `if (options.bidirectional)`

Reply:
Sorry, I misunderstood, this can be set to `false` i presume

Reply:
yes, these can be explicitly set to false.